src/richie/apps/courses/models/course.py

```diff
@@ -572,63 +621,55 @@ def copy_translations(self, oldinstance, language=None):
             translation_object.master = self
             translation_object.save()
 
-    # pylint: disable=arguments-differ
-    def save(self, *args, **kwargs):
+    def mark_course_dirty(self):
```

Comment:
I'm curious to know why you chosse `dirty` word ? It's ugly 😅

Reply:
Ah :man_shrugging:  This is not my choice... https://github.com/django-cms/django-cms/blob/develop/cms/models/titlemodels.py#L86